src/reducers/comments.spec.tsx

```diff
@@ -176,4 +227,127 @@ describe(__filename, () => {
       });
     });
   });
+
+  describe('manageComment', () => {
+    const _manageComment = (params = {}) => {
+      return manageComment({
+        _createOrUpdateComment: jest
+          .fn()
+          .mockResolvedValue(createFakeExternalComment()),
+        addonId: 1,
+        cannedResponseId: undefined,
+        comment: 'Example of a comment',
+        commentId: undefined,
+        fileName: null,
+        line: null,
+        versionId: 432,
+        ...params,
+      });
+    };
+
+    it('calls createOrUpdateComment', async () => {
+      const _createOrUpdateComment = jest
+        .fn()
+        .mockResolvedValue(createFakeExternalComment());
+      const addonId = 123;
+      const cannedResponseId = undefined;
+      const comment = 'A comment on a file';
+      const commentId = undefined;
+      const fileName = 'manifest.json';
+      const line = 543;
+      const versionId = 321;
+
+      const { store, thunk } = thunkTester({
+        createThunk: () =>
+          _manageComment({
+            _createOrUpdateComment,
+            addonId,
+            cannedResponseId,
+            comment,
+            fileName,
+            line,
+            versionId,
+          }),
+      });
+
+      await thunk();
+
+      expect(_createOrUpdateComment).toHaveBeenCalledWith({
+        addonId,
+        apiState: store.getState().api,
+        cannedResponseId,
+        comment,
+        commentId,
+        fileName,
+        line,
+        versionId,
+      });
+    });
+
+    it('dispatches beginSaveComment()', async () => {
+      const versionId = 1;
```

Comment:
Could you combine these into a `params` object, like you did for some of the other tests in this file, and then just call `_manageComment(params)` and `toHaveBeenCalledWith(actions.beginSaveComment(params))`?